src/subprocess-posix.cc

```diff
@@ -21,6 +21,7 @@
 #include <unistd.h>
 #include <stdio.h>
 #include <string.h>
+#include <sys/select.h>
```

Comment:
why?

Reply:
to get FD_* definitions from sys/select.h

https://github.com/ninja-build/ninja/blob/master/src/subprocess-posix.cc#L334
https://pubs.opengroup.org/onlinepubs/007904975/basedefs/sys/select.h.html

Reply:
I checked again, and it seems to work fine without it. Must be a restover. It's now removed.